packages/tfm-ror52-rubygem-thor/tfm-ror52-rubygem-thor.spec

```diff
@@ -0,0 +1,95 @@
+# Generated from thor-0.20.0.gem by gem2rpm -*- rpm-spec -*-
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name thor
+
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.20.0
+Release: 1%{?dist}
+Summary: Thor is a toolkit for building powerful command-line interfaces
+Group:   Development/Languages
+License: MIT
+URL:     http://whatisthor.com/
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+BuildRequires: %{?scl_prefix}build
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel >= 1.3.5
+BuildRequires: %{?scl_prefix_ruby}ruby >= 1.8.7
+BuildArch: noarch
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+
+%{?scl:Requires: %{?scl_prefix}runtime}
+
+%description
+Thor is a toolkit for building powerful command-line interfaces.
+
+
+%package doc
+Summary:   Documentation for %{name}
+Group:     Documentation
+Requires:  %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}
+
+%{?scl:scl enable %{scl} - << \EOF}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%{?scl:EOF}
+
+%build
+# Create the gem as gem install only works on a gem file
+%{?scl:scl enable %{scl} - << \EOF}
+gem build %{gem_name}.gemspec
+%{?scl:EOF}
+
+# %%gem_install compiles any C extensions and installs the gem into ./%%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+mkdir -p ./opt/rh/%{scl_ruby}/root/usr/bin
```

Comment:
Should this line be inside the `scl enable`? It'll break on non-SCL anyway.

Reply:
I find that being inside masks this command when examining build log. Also, we don't do non-SCL builds so I am good with it here.